tests/s3/test_multipart_getput.py

```diff
@@ -316,19 +322,28 @@ def test_multipart_upload_test_28538(self):
         self.log.info("Listed parts of multipart upload: %s", res[1])
         self.log.info(" Complete the multipart with first and last part upload")
         uploaded_parts[1].reverse()
-        try:
-            resp = self.s3_mpu_test_obj.complete_multipart_upload(mpu_id, uploaded_parts[1],
+        if S3_ENGINE_RGW == CMN_CFG["s3_engine"]:
```

Comment:
Are the multi part upload APIs different for RGW from standard multipart low level APIs?

Reply:
No, they are same